ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/runtime/CallbackController.java

```diff
@@ -123,11 +126,22 @@ CaseDetails convertCaseDetails(uk.gov.hmcts.reform.ccd.client.model.CaseDetails
   @SneakyThrows
   CaseDetails convertCaseDetails(uk.gov.hmcts.reform.ccd.client.model.CaseDetails ccdDetails,
                                  String caseType) {
-    String json = mapper.writeValueAsString(ccdDetails);
+
     if (!caseTypeToJavaType.containsKey(caseType)) {
       log.warn("Handler not found for " + caseType);
       throw new ResponseStatusException(HttpStatus.NOT_FOUND, "Handler not found for " + caseType);
     }
+
+    if (null != ccdDetails) {
+      Map<String, Object> migratedData = caseTypeToConfig.get(caseType).getPreEventHooks()
```

Comment:
caseTypeToConfig.get(caseType) -> should we put a null check around this ?

Reply:
it's covered by line 130